spec/features/food_items/new_food_item_spec.rb

```diff
@@ -0,0 +1,152 @@
+
+          expect(page).to have_content("Name can't be blank")
+        end
+
+        scenario 'doesn`t create a new food item' do
+          expect{ submit_form }.not_to change(FoodItem, :count)
+        end
+      end
+
+      context 'without item price' do
+        let(:food_item_params) { attributes_for(:food_item).merge(price: nil) }
+
+        scenario 'show form error' do
+          submit_form
+
+          expect(page).to have_content("Price can't be blank")
+        end
+
+        scenario 'doesn`t create a new food item' do
+          expect{ submit_form }.not_to change(FoodItem, :count)
+        end
+      end
+
+      context 'without item description' do
+        let(:food_item_params) { attributes_for(:food_item).merge(description: '') }
+
+        scenario 'show form error' do
+          submit_form
+
+          expect(page).to have_content("Description can't be blank")
+        end
+
+        scenario 'doesn`t create a new food item' do
+          expect{ submit_form }.not_to change(FoodItem, :count)
+        end
+      end
+
+      context 'without item meal_type' do
+        let(:food_item_params) { attributes_for(:food_item).merge(meal_type: '') }
+
+        scenario 'show form error' do
+          submit_form
+
+          expect(page).to have_content("Meal type can't be blank")
+        end
+
+        scenario 'doesn`t create a new food item' do
+          expect{ submit_form }.not_to change(FoodItem, :count)
+        end
+      end
+
+      context 'name is too long' do
+        let(:food_item_params) { attributes_for(:food_item).merge(name: Faker::Lorem.paragraph)}
+
+        scenario 'show form error' do
+          submit_form
+
+          expect(page).to have_content("Name is too long (maximum is 20 characters)")
+        end
+
+        scenario 'doesn`t create a new food item' do
+          expect{ submit_form }.not_to change(FoodItem, :count)
+        end
+      end
+
+      context 'name is too short' do
+        let(:food_item_params) { attributes_for(:food_item).merge(name: 'ab')}
+
+        scenario 'show form error' do
+          submit_form
+
+          expect(page).to have_content("Name is too short")
+        end
+
+        scenario 'doesn`t create a new food item' do
+          expect{ submit_form }.not_to change(FoodItem, :count)
+        end
+      end
+
+      context 'price is not digit' do
+        let(:food_item_params) { attributes_for(:food_item).merge(price: 'aabc')}
+
+        scenario 'show form error' do
+          submit_form
+
+          expect(page).to have_content("Price is not a number")
+        end
+
+        scenario 'doesn`t create a new food item' do
+          expect{ submit_form }.not_to change(FoodItem, :count)
+        end
+      end
+    end
+  end
+
+  context 'user is not admin' do
+    let(:customer) { create(:user) }
+
+    before do
+      create(:admin)
+
+      sign_in customer
+
+      visit new_food_item_path
+    end
+
+      scenario 'no permission' do
```

Comment:
It is common to start your context description with when, and to have a verb in your scenario/it description. E.g.
 `when user is not an admin`
`  shows 'no permission' message`